codegen/src/test/resources/q3/Q3.json

```diff
@@ -5,101 +5,109 @@
       "ForeignRelation": "Lineitem"
     },
     {
-      "PrimaryRelation": "Orders",
-      "ForeignRelation": "Customer"
+      "PrimaryRelation": "Customer",
+      "ForeignRelation": "Orders"
     }
   ],
-  "LineitemProcessFunction": {
-    "name": "Q3lineitem",
-    "relation": "lineitem",
-    "this_key": [
-      "orderkey"
-    ],
-    "next_key": [
-      "orderkey"
-    ],
-    "child_nodes": 1,
-    "is_Root": true,
-    "is_Last": true,
-    "rename_attribute": null,
-    "select_conditions": {
-      "operator": "&&",
-      "value1": {
-        "left_field": {
-          "type": "attribute",
-          "name": "l_shipdate"
-        },
-        "right_field": {
-          "type": "constant",
-          "var_type": "Date",
-          "value": "1995-03-15"
-        },
-        "operator": ">"
+  "RelationProcessFunction": {
+    "LineitemProcessFunction": {
+      "name": "Q3lineitem",
+      "relation": "lineitem",
+      "this_key": [
+        "orderkey"
+      ],
+      "next_key": [
+        "orderkey"
+      ],
+      "child_nodes": 0,
+      "is_Root": true,
+      "is_Last": true,
+      "rename_attribute": null,
+      "select_conditions": {
+        "operator": "&&",
+        "value1": {
```

Comment:
The value1 can make as an array named "value", which allows arbitrary number of values.